lib/task-data/tasks/install-coreos.js

```diff
@@ -11,7 +11,8 @@ module.exports = {
         comport: 'ttyS0',
         hostname: 'coreos-node',
         installDisk: '/dev/sda',
-        completionUri: 'pxe-cloud-config.yml'
+        completionUri: 'pxe-cloud-config.yml',
+        repo: '{{api.server}}/coreos'
```

Comment:
Will this structure for the repo work as a task definition if we want to support a secondary proxy that's providing access to the data (i.e. an installed instance of `squid` or such)?